sw/device/lib/arch/device_sim_dv.c

```diff
@@ -40,11 +42,13 @@ const uint32_t kUartTxFifoCpuCycles =
 const uint32_t kAstCheckPollCpuCycles =
     CALCULATE_AST_CHECK_POLL_CPU_CYCLES(kClockFreqCpuHz);
 
-// Defined in `hw/top_earlgrey/dv/env/chip_env_pkg.sv`
-const uintptr_t kDeviceTestStatusAddress = 0x411f0080;
+// TODO: Need to find a way to feed the RV_CORE_IBEX base in.
+const uintptr_t kDeviceTestStatusAddress =
+    0x411f0000 + RV_CORE_IBEX_DV_SIM_WINDOW_REG_OFFSET;
 
-// Defined in `hw/top_earlgrey/dv/env/chip_env_pkg.sv`
-const uintptr_t kDeviceLogBypassUartAddress = 0x411f0084;
+// TODO: Need to find a way to feed the RV_CORE_IBEX base in.
```

Comment:
nit: this should come from a _top_ header, right?

Reply:
It should - let me change that.
This is one example of multitop incompatibility.
I wonder whether we should rename the header to something generic (e.g. `toplevel.h`) so that this can be handled by the build system mostly.